README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
+
+
+* _data manipulation_
+* _data visualization_
```

Comment:
Merit: the use of headline and bullet points is great. You have specified your points here. 
Improvement: I suggest you to add some subheading here to get your page more organized.